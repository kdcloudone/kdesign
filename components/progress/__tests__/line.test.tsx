import React from 'react'
import { render, mount } from 'enzyme'
import mountTest from '../../../tests/shared/mountTest'
import Line from '../line'

describe('Progress Line', () => {
  mountTest(Line)

  it('renders correctly', () => {
    expect(render(<Line />)).toMatchSnapshot()
  })

  it('should have displayName static property', () => {
    const wrapper = mount(<Line />)
    expect((wrapper.type() as any).displayName).toBe('Line')
  })
})
