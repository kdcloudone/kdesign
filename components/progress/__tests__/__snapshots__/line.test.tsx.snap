// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress Line renders correctly 1`] = `
<div
  class="undefined-outer"
>
  <div
    class="undefined-inner"
  >
    <div
      class="undefined-bg"
      style="width:0%"
    />
  </div>
</div>
`;
